src/deploy/hosting/convertConfig.ts

```diff
@@ -49,17 +49,16 @@ export function findEndpointForRewrite(
   targetBackend: backend.Backend,
   id: string,
   region: string | undefined
-): backend.Endpoint | undefined {
+): [backend.Endpoint | undefined, boolean] {
```

Comment:
I'd maybe prefer returning an object rather than a tuple like this. Object can be way better documented (and generally follow style guide for T/Js much better).

Also, on that note, I'm not sure what the bool is for. Can you add some docs for this?

Reply:
Done. The bool is for "we saw this function somewhere", which means it's a firebase function and we complain if things don't look right.